www/views/minigame.php

```diff
@@ -190,18 +389,70 @@ function updateGame() {
       ctx.lineTo(laser.x, laser.y);
       ctx.stroke();
     });
+    //game paused overlay
+    if (gameState == 1) {
+      ctx.fillStyle = "rgb(32,32,32,0.4)";
+      ctx.fillRect(0 , 0, canvas.width, canvas.height);
+      ctx.fillStyle = "white";
+      ctx.font = "65px Arial bold";
+      ctx.fillText("GEPAUZEERD", (canvas.width / 2) - ctx.measureText("GEPAUZEERD").width / 2, (canvas.height / 2) - 100);
+    }
+    //game over overlay
+    if (gameState == 2) {
+      ctx.fillStyle = "rgb(32,32,32,0.4)";
+      ctx.fillRect(0 , 0, canvas.width, canvas.height);
+      ctx.fillStyle = "white";
+      ctx.fillRect((canvas.width / 2) - 102, (canvas.height / 2) + 148, 204, 54);
+      ctx.fillStyle = "gray";
+      ctx.fillRect((canvas.width / 2) - 100, (canvas.height / 2) + 150, 200, 50);
+      ctx.fillStyle = "white";
+      ctx.font = "65px Arial bold";
+      ctx.fillText("GAME OVER", (canvas.width / 2) - ctx.measureText("GAME OVER").width / 2, (canvas.height / 2) - 160);
```

Comment:
Werken die fonts bij jou wel? Want bij mij is dit scherm een ander font dan de informatie linksboven:

![Screenshot from 2025-04-09 15-33-15](https://github.com/user-attachments/assets/14d0fcce-0a5c-4f46-80cd-f85b946434c7)